src/store/channels-list/saga.ts

```diff
@@ -110,8 +104,10 @@ export function* fetchConversations() {
     chatClient,
     chatClient.getConversations,
   ]);
+  const otherMembersOfConversations = conversations.map((c) => c.otherMembers);
```

Comment:
Are we sure this works? This would create an array of arrays.... doesn't seem like the user presence function handles that.

Reply:
I'm also worried that at this point the member objects haven't been mapped to Zero users yet? So, perhaps this should happen after that call.

Reply:
oops yes I was supposed to flatten! Updated the the order as well. Thanks!